README.md

```diff
@@ -5,11 +5,12 @@ Respond to short response questions in clear, concise sentences directly within
 
 ### Short Response Questions
 **1. What is recursion?**
-
+Recrsion is an algorithm in which it calls it self in order to break down into a smaller poblem for it to be solved in an elegant and readable way
 **2. What type of problems are best solved using recursion?**
+The best type of problem  to solve with recursion are those that have a repeating solution
```

Comment:
What other similarities in terms of the algorithm do these recursive solutions have?